openslides_backend/action/motion/reset_state.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict
+
+from ...models.models import Motion
+from ...shared.exceptions import ActionException
+from ...shared.patterns import Collection, FullQualifiedId
+from ...shared.schema import schema_version
+from ..generics import UpdateAction
+from ..register import register_action
+
+reset_schema = {
+    "$schema": schema_version,
+    "title": "Reset motion state schema",
+    "description": "An array of objects with motion id",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {"id": {"type": "integer"}},
+        "required": ["id"],
+        "additionalProperties": False,
+    },
+    "minItems": 1,
+    "uniqueItems": True,
+}
+
+
+@register_action("motion.reset_state")
+class MotionResetStateAction(UpdateAction):
+    """
+    Reset motion state action.
+    """
+
+    schema = reset_schema
+    model = Motion()
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Set state_id to motion_state.first_state_of_workflow_id.
+        """
+        motion = self.database.get(
+            FullQualifiedId(Collection("motion"), instance["id"]), ["state_id"]
+        )
+        if motion.get("state_id"):
```

Comment:
Throw an error in the else part, a motion must always have a state.

Reply:
Done.